build.gradle

```diff
@@ -160,6 +160,7 @@ allprojects {
 
 configurations {
     zipArchive
+    secureIntegTestPluginArchive
```

Comment:
In the future, I think we could make the configurations follow an inheritance hierarchy:
```
zipArchive
compileTimeZipArchive.extendsFrom zipArchive // for k-NN
```

but I think for now this is fine.